tasks/collector.py

```diff
@@ -0,0 +1,108 @@
+import os
+import platform
+import subprocess
+import tempfile
+import urllib.request
+
+from invoke.exceptions import Exit
+from invoke.tasks import task
+
+from tasks.go import generate_licenses, tidy
+from tasks.libs.common.color import Color, color_message
+
+LICENSE_HEADER = """// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+"""
+OCB_VERSION = "v0.103.1"
+
+
+@task(post=[tidy, generate_licenses])
+def generate(ctx):
+    arch = platform.machine()
+    system = platform.system()
+    base_url = (
+        f"https://github.com/open-telemetry/opentelemetry-collector/releases/download/cmd%2Fbuilder%2F{OCB_VERSION}/"
+    )
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
**💬 suggestion**
This could be a const